From: =?utf-8?q?C=C3=A9dric_Boutillier?= <boutil@debian.org>
Date: Mon, 20 Apr 2020 01:40:06 +0200
Subject: remove bundler setup

Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=955673
Last-Update: 2020-04-20

this prevents rake to load correctly in a clean build environment
---
 Rakefile | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/Rakefile b/Rakefile
index 9ee81cc..fc1119b 100644
--- a/Rakefile
+++ b/Rakefile
@@ -1,9 +1,3 @@
-begin
-  require 'bundler/setup'
-  Bundler::GemHelper.install_tasks
-rescue Exception
-end
-
 require 'rake/testtask'
 
 desc 'Run Slim benchmarks! (default parameters slow=false)'
